### PR TITLE
05B Shared elements

### PR DESCRIPTION
**Pull request by pedrovgs**
Remove GlobalPositionTransition and replace it with the already implemented transitions.

### CODE REVIEW

**Review by davideme — APPROVED** [review]
👍 
Let see how @Serchinastico like to handle de merges at this is not the tipical project.

**Comment by Serchinastico** [comment]
I'm going to review it and update the branch to point to the previous branch so that we can show different solutions to the same problem as we've been doing with others. Thanks Pedro!

File: app/src/main/java/com/karumi/androidanimations/sharedelements/SharedElementsExerciseTransitionFragment.kt [hunk 1]
```diff
@@ -57,7 +56,14 @@ class SharedElementsExerciseTransitionFragment : BaseFragment() {
 
         val enterSharedElementTransition
             get() = TransitionSet()
-                .addTransition(GlobalPositionTransition().accelerated.withArc)
```
Comment (Serchinastico) [inline app/src/main/java/com/karumi/androidanimations/sharedelements/SharedElementsExerciseTransitionFragment.kt]: I find it interesting that it looks the same as the exit transition xD

**Review by Serchinastico — APPROVED** [review]
Nicely done!

File: app/src/main/java/com/karumi/androidanimations/sharedelements/SharedElementsExerciseTransitionFragment.kt (hunk 1 above)
Comment (pedrovgs) [reply app/src/main/java/com/karumi/androidanimations/sharedelements/SharedElementsExerciseTransitionFragment.kt]: Makes sense! With the transitions API you only specify how the change between the values should animate in terms of timing and what values should animate but not the values 😃

**Pull request closed without merging**